js.go

```diff
@@ -3646,3 +3656,22 @@ func (st *StorageType) UnmarshalJSON(data []byte) error {
 	}
 	return nil
 }
+
+// Length of our hash used for named consumers.
+const nameHashLen = 8
+
+// Computes a hash of 8 characters for the name.
+func getHash(name string) string {
```

Comment:
Since we don't use any other size would collapse into one function with fixed size of 8 for now.

Reply:
sounds good

Reply:
updated to a single function